app/graphql/types/mutation_type.rb

```diff
@@ -1,5 +1,6 @@
 class Types::MutationType < Types::BaseObject
   field :pro, Types::Mutations::ProMutation, null: false
+  field :account, Types::Mutations::AccountMutation, null: true
   field :anime, Types::Mutations::AnimeMutation, null: true
   field :episode, Types::Mutations::EpisodeMutation, null: true
   field :library_entry, Types::Mutations::LibraryEntryMutation, null: true
```

Comment:
just want to leave this here, should we maybe alphabetize these?

Reply:
I was thinking that too, honestly